clients/youtube_producer/js/yt_auth/yt_authDirective.js

```diff
@@ -41,24 +28,19 @@ module.exports = function ($window, $http, $interval, $log) {
 					if(e.originalEvent){
 						e = e.originalEvent;
 					}
-					if (e.origin !== AUTH_SERVER) {
+					if (e.origin !== config.authServer) {
 							$log.log('domain does not match!');
```

Comment:
Is it only the domain that has to match? If I remember correctly, you told me that the port also needs to match. What about `Requesting(?) origin does not match`